recipes/colour-demosaicing/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "colour-demosaicing" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/colour-demosaicing-{{ version }}.tar.gz
+  sha256: 8886a101f40eee8056a2762732f783304ba37026b638de9fb16fa8b8a10ac039
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
+    - poetry
+  run:
+    - colour-science >=0.4.0
+    - imageio >=2,<3
+    - numpy >=1.19,<2
+    - python >=3.6,<3.11
+    - scipy >=1.5,<2
+    - typing-extensions >=4,<5
+
+test:
+  imports:
+    - colour_demosaicing
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.colour-science.org/
```

Comment:
Shouldn't the home be https://www.colour-science.org/colour-demosaicing/

Reply:
Can you also add a link to their github https://github.com/colour-science/colour-demosaicing and to their documentation, e.g., https://colour-demosaicing.readthedocs.io/en/latest/reference.html?

Reply:
okay I will add those using dev_url and doc_url but if there is a different way to do it just make suggested changes